bin/sparkling-env.sh

```diff
@@ -47,7 +47,9 @@ FAT_JAR="sparkling-water-assembly_$SCALA_VERSION-$VERSION-all.jar"
 FAT_JAR_FILE="$TOPDIR/assembly/build/libs/$FAT_JAR"
 major_version=`echo $VERSION | cut -d . -f 1,2`
 version_without_snapshot=`echo $VERSION | cut -d - -f 1`
-PY_EGG="h2o_pysparkling_${major_version}-${version_without_snapshot}-py2.7.egg"
+
+PYTHON_VERSION=`python -c 'import sys; print(".".join(str(x) for x in sys.version_info[0:2]))'`
```

Comment:
@jakubhava We don't have a `pysparkling` script for Windows, though and we're not setting it (the egg file location) in the Windows version - should we? Was it not done before for any reason?

Added a warn message in the script, not failing the script though as some uses of it don't need PySparkling.